src/defaults/config/.linear/themes/dark.css

```diff
@@ -0,0 +1,220 @@
+html,
+body {
+    height: 100%;
+}
+
+html {
+    font-size: 10px;
+    min-width: 0;
+    overflow: hidden;
+}
+
+body {
+    min-width: 0;
+    font-family: 'Open Sans';
+    font-weight: 300;
+    overflow: auto;
+}
+
+.ruler-mask {
+    -webkit-app-region: drag;
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    background-color: #888;
+    opacity: 0.3;
+}
+
+.ruler {
+    -webkit-user-select: none;
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    transition: all .5s ease-out;
+}
+
+.ruler:hover {
+    /*background-color: rgba(0, 0, 0, 0.4);*/
+}
+
+.ruler__inner {
+    position: relative;
+    width: 100%;
+    height: 100%;
+}
+
+.ruler__inner__border {
+    position: absolute;
+}
+
+.ruler__inner__border--left {
+    left: 5em;
+    height: 100%;
+    border-left: 1px solid black;
+}
+
+.ruler__inner__border--right {
+    right: 5em;
+    height: 100%;
+    border-right: 1px solid black;
+}
+
+.ruler__inner__border--top {
+    top: 5em;
+    width: 100%;
+    border-top: 1px solid black;
+}
+
+.ruler__inner__border--bottom {
+    bottom: 5em;
+    width: 100%;
+    border-bottom: 1px solid black;
+}
+
+.ruler__inner__info {
+    background-color: rgba(34,34,34,0.3);
+    position: absolute;
+    bottom: .6em;
+    left: 4.9em;
+    padding: 3px 5px 3px 3px;
+    font-size: 1.1em;
+    color: white;
+    z-index: 99;
+}
+
+@media (max-width:152px) {
+    .ruler__inner__info {
+        left: 1em;
+    }
+}
+
+.dark-theme .ruler__inner__info {
```

Comment:
I was sure that I had removed these, because I did, so I was confused for a second, but then I realized that I only removed them from the unused `ruler.css` file and not from the `universal.css` file. Whoops! :tomato: